sql/moz-fx-data-shared-prod/regrets_reporter_derived/regrets_reporter_summary_v1/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+friendly_name: Regrets Reporter Summary
+description: |
+  Measure usage of the regrets reporter addon
+  See https://github.com/mozilla/bigquery-etl/issues/2572
+owners:
+- kignasiak@mozilla.com
+labels: {}
+scheduling:
+  dag_name: bqetl_regrets_reporter_summary
+  task_name: regrets_reporter_summary__v1
+bigquery:
+  time_partitioning:
+    field: date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - country
+    - browser
+workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:mozilla-confidential
```

Comment:
I'm pretty sure this should not be `mozilla-confidential` since we want anyone to be able to access this particular table. @jklukas can give better guidance than me here though.

Reply:
`workgroup:mozilla-confidential` corresponds to  "Mozilla Confidential - Staff and NDA'd Mozillians Only" per https://wiki.mozilla.org/Security/Data_Classification

It's the right thing here.